portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -16,17 +16,53 @@
 
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
+import java.util.*;  
```

Comment:
The google style guide forbids star imports. Can this be more granular?

Reply:
Ack.